__tests__/examples/webpack.skpm.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path');
+
+const aliasedModules = [
```

Comment:
could we get this list from the package.json? Just to make sure nothing is missing